assets/js/aplus.js

```diff
@@ -503,7 +503,7 @@ $(function() {
                         modal.trigger("opened.aplus.modal");
                         // render math in the modal
                         if (typeof window.MathJax !== "undefined") {
-                            window.MathJax.Hub.Queue(["Typeset", window.MathJax.Hub, modal.get(0)]);
+                            $.fn.aplusTypesetMath(modal.get(0));
```

Comment:
```suggestion
                            modal.aplusTypesetMath();
```
After changing the plugin definition, this line would be something like that.